about.php

```diff
@@ -4,12 +4,13 @@
 <head>
 	<meta charset="UTF-8">
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>About Us</title>
 	<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
 	<link rel="stylesheet" href="css/main.css" />
-	<title>About Us</title>
 </head>
 
 <body>
+	<?php include 'navBar.php'; ?>
```

Comment:
you have a file name navbar.php but this is looking for `navBar.php` so it doesnt work

Reply:
This was intentional, my original plan was to work on both navbar and footer on the same branch but I will update the file name to reflect the appropriate changes. As of the navbar.php vs navBar.php I didn't make any changes to that not sure why its not showing the right file name.

Reply:
![Desktop screensh](https://user-images.githubusercontent.com/60706674/111855110-4cb38600-88f9-11eb-9ad8-2ad64c85d0ad.png)

Reply:
I'm not sure what you're showing me in that screenshot... you made a file called `navbar.php` and tried to include `navBar.php`. Based on the commit history you made both changes... I'm confused what you mean here...

Reply:
if you look at 2e4ab7e23e9708c63ce4b5bd37e8dbb6e870e55d you'll see the filename you created doesn't match the one you're trying to refer to.

Reply:
Yup, I see what you are saying. I will the issue.